packages/qgrid-ngx/src/lib/column/column.component.ts

```diff
@@ -142,7 +142,9 @@ export class ColumnComponent implements OnInit, OnDestroy, OnChanges {
 
     if (withKey) {
       if (this.parentHost) {
-        this.parentHost.column.children.push(column);
+        if (this.parentHost.column.children) {
+          this.parentHost.column.children.push(column);
```

Comment:
actually you can replace these lines with 
`this.parentHost.column.children?.push(column);`